core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -128,7 +139,49 @@
     text-align: right;
     vertical-align: middle;
   }
+  .audio-header-control-buttons {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 auto;
+    width: 560px;
+  }
+  .oppia-replay-white-circle {
+    cursor: pointer;
+    display: inline-block;
+    font-size: 1.4em;
+    height: 36px;
+    padding: 5px;
+    position: relative;
+    vertical-align: middle;
+    width: 36px;
+  }
 
+  .audio-undo-icon {
+    color: #fff;
+    left: 50%;
+    padding: 0;
+    position: absolute;
+    top: 7px;
+    transform: translateX(-43%);
+  }
+  .audio-undo-icon-forward {
+    transform: scaleX(-1) translateX(43%);
```

Comment:
make sure this is compatible with all other browsers (and old browsers)

Reply:
I think we are good.
https://developer.mozilla.org/en-US/docs/Web/CSS/transform